src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -85,19 +84,17 @@ const WorkspaceTabs = ({
   ]
   return h(Fragment, [
     h(TabBar, {
-      'aria-label': 'workspace menu',
+      'aria-label': 'Workspace Navigation Tabs',
```

Comment:
Both the tabs and the workspace action menu were named "Workspace menu", which I found to be confusing.